api/scholarqa/rag/multi_step_qa_pipeline.py

```diff
@@ -82,20 +82,16 @@ def make_prompt(query: str, paper_paper_quotes_dict: Dict[str, str]) -> str:
 
         user_prompt = make_prompt(query, per_paper_summaries)
         try:
+            #params for reasoning mode: max_completion_tokens=4096, max_tokens=4096+1024, reasoning_effort="low"
             response = llm_completion(user_prompt=user_prompt,
                                       system_prompt=sys_prompt, fallback=None, model=self.llm_model,
                                       max_tokens=4096,
-                                      response_format={"response_schema": ClusterPlan.model_json_schema(
-                                          ref_template="/$defs/{model}")}
+                                      response_format= ClusterPlan
                                       )
+            return json.loads(response.content), response
         except Exception as e:
-            logger.warning(f"Error while clustering with {self.llm_model}: {e}, trying to fall back to GPT-4o.")
-            response = llm_completion(user_prompt=user_prompt,
-                                      system_prompt=sys_prompt, fallback=None, model=GPT_4o,
-                                      max_tokens=4096,
-                                      response_format=ClusterPlan
-                                      )
-        return json.loads(response.content), response
```

Comment:
This fix was to get around breaking code in litellm at the time, which has been fixed in the latest version. https://github.com/BerriAI/litellm/issues/7755